### PR TITLE
[fptplay] Improve extractor

### PR DESCRIPTION
**Pull request by hatienl0i261299**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Fix bug get title, description and video info by episode.

### CODE REVIEW

**Comment by pukkandan** [comment]
1. Unnecessary escapes in regex
2. non-fatal fields are not handled correctly
3. indentations that don't make any sense

I have pushed the necessary changes for this PR because that is honestly easier than reviewing each issue. But don't expect me to do that for future PRs

File: yt_dlp/extractor/fptplay.py [hunk 1]
```diff
@@ -2,11 +2,14 @@
 from __future__ import unicode_literals
 
 import hashlib
+import re
```
Comment (pukkandan) [inline yt_dlp/extractor/fptplay.py:5]:
```suggestion
```

**Review by pukkandan — APPROVED** [review]
pls confirm everything's still working

File: yt_dlp/extractor/fptplay.py [hunk 2]
```diff
@@ -7,12 +7,14 @@
...
 from ..utils import (
+    clean_html,
     join_nonempty,
+    strip_or_none,
 )
 
 
 class FptplayIE(InfoExtractor):
-    _VALID_URL = r'https?://fptplay\.vn/(?P<type>xem-video)/[^/]+\-(?P<id>\w+)(?:/tap-(?P<episode>[^/]+)?/?(?:[?#]|$)|)'
+    _VALID_URL = r'https?://fptplay\.vn/xem-video/[^/]+\-(?P<id>\w+)(?:/tap-(?P<episode>[^/]+)?/?(?:[?#]|$)|)'
```
Comment (pukkandan) [inline yt_dlp/extractor/fptplay.py:17]: shouldn't this be `(?P<episode>\d+)`

Comment (pukkandan) [reply yt_dlp/extractor/fptplay.py:17]: and about this?

**Comment by hatienl0i261299** [comment]
@pukkandan Is my PR okay, Can we merge?

**Pull request merged**